packages/synapse-react-client/src/components/widgets/facet-nav/PlotsContainer.test.tsx

```diff
@@ -64,7 +64,6 @@ function init(props?: PlotsContainerProps) {
 }
 
 describe('facets display hide/show', () => {
-  failOnConsole()
```

Comment:
Upgrading to React 18.3.1 caused a new console.error because react-sizeme is using an API that was removed in React 19.

Created a ticket: PORTALS-3592